src/main/java/pikabot/Ui.java

```diff
@@ -0,0 +1,144 @@
+                + "\n" + INDENTATION + LINE);
+    }
+
+    /**
+     * Prints out list of tasks.
+     *
+     * @param taskList TaskList containing tasks to print.
+     */
+    public static void printListOfTasks(TaskList taskList) {
+        System.out.println(INDENTATION + LINE);
+        System.out.println(INDENTATION + "Here are the tasks in your list:");
+
+        int taskNumber = 1;
+        int length = taskList.noOfTasks();
+
+        while (taskNumber <= length) {
+            System.out.println(INDENTATION + taskNumber + "."
+                    + taskList.get(taskNumber - 1));
+            taskNumber++;
+        }
+        System.out.println(INDENTATION + LINE);
+    }
+
+    /**
+     * Prints out list of tasks containing a specified keyword.
+     *
+     * @param taskList TaskList containing tasks with the specified keyword.
+     * @param keyword Keyword to search for that is given by user.
+     */
+    public static void printListOfMatchedTasks(TaskList taskList, String keyword) {
+        System.out.println(INDENTATION + LINE);
+        System.out.println(INDENTATION + "Here are the tasks containing \"" + keyword + "\" in your list:");
+
+        int taskNumber = 1;
+        int length = taskList.noOfTasks();
+
+        while (taskNumber <= length) {
+            System.out.println(INDENTATION + taskNumber + "."
+                    + taskList.get(taskNumber - 1));
+            taskNumber++;
+        }
+
+        System.out.println(INDENTATION + LINE);
+    }
+
+    /**
+     * Prints out indication that a task has been marked as done.
+     *
+     * @param task Task that has been marked as done.
+     */
+    public static void indicateMarked(Task task) {
+        System.out.println(INDENTATION + LINE);
+        System.out.println(INDENTATION + "Nice! I've marked this task as done:");
+        System.out.println(INDENTATION + task);
+        System.out.println(INDENTATION + LINE);
+    }
+
+    /**
+     * Prints out indication that a task has been unmarked.
+     *
+     * @param task Task that has been unmarked.
+     */
+    public static void indicateUnmarked(Task task) {
+        System.out.println(INDENTATION + LINE);
+        System.out.println(INDENTATION + "Ok, I've marked this task as not done yet:");
+        System.out.println(INDENTATION + task);
+        System.out.println(INDENTATION + LINE);
+    }
+
+    /**
+     * Prints out indication that a task has been added.
+     *
+     * @param task Task that has been added.
+     * @param taskList TaskList that task has been added to.
+     */
+    public static void indicateAddedTask(Task task, TaskList taskList) {
+        System.out.println(INDENTATION + LINE + "\n"
+                + INDENTATION + "Got it. I've added this task:" + "\n"
+                + INDENTATION + "  " + task + "\n"
+                + INDENTATION + "Now you have " + taskList.noOfTasks() + " tasks in the list."
+                + "\n" + INDENTATION + LINE);
```

Comment:
Might be nit picking here, but if the "\n" can be on the previous line, the readability will be 💯

Reply:
Provided it doesn't exceed the character limit per line